index.html

```diff
@@ -27,7 +27,164 @@ <h1 id="name">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/24.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/25.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/26.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="container3">
+    <h1 id="about">
+      About
+    </h1>
+    <p id="myintro">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+       sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut enim ad minim veniam, 
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+    </p>
+    <div class="button">
+      <div class="btn" id="unique">Get my CV</div> 
+      <div id="ic">
+        <a href="#"><img src="img/ic_download.png" alt=""></a>
+      </div>
+    </div>
+  </section>
+
+  <section class="container4">
+    <h1 class="title-about">Languages</h1>
```

Comment:
**OPTIONAL** - For better SEO it is not advisable to have more than one `h1` tag in your HTML page. Please remove all `h1` tags on your page, and be sure you have at least one, but no more than one `h1` tag on your HTML page 🙏